app/api/tests/data/example.csv

```diff
@@ -1,4 +1,4 @@
-text,label
-AAA,Positive
-BBB,Positive
+text,label,meta
+AAA
+BBB,Positive,The following is meta data
```

Comment:
I tested the case of `text` and `meta` but it failed.

```
text,label,meta
text1,,meta1
text2,,meta2
```

Reply:
This now works and I added this specific to the testcase.